diff --git a/src/nvim/fileio.c b/src/nvim/fileio.c
index c3cd23c5f..94346b6b9 100644
--- a/src/nvim/fileio.c
+++ b/src/nvim/fileio.c
@@ -2620,12 +2620,12 @@ int buf_write(buf_T *buf, char_u *fname, char_u *sfname, linenr_T start, linenr_
     /*
      * Check if the timestamp hasn't changed since reading the file.
      */
-    if (overwriting) {
-      retval = check_mtime(buf, &file_info_old);
-      if (retval == FAIL) {
-        goto fail;
-      }
-    }
+    /* if (overwriting) { */
+    /*   retval = check_mtime(buf, &file_info_old); */
+    /*   if (retval == FAIL) { */
+    /*     goto fail; */
+    /*   } */
+    /* } */
   }
 
 #ifdef HAVE_ACL
